src/creature.cpp

```diff
@@ -289,9 +289,17 @@ Creature *Creature::auto_find_hostile_target( int range, int &boo_hoo, int area
                 path_to_target.insert( path_to_target.begin(), pos() );
 
                 // Getting point on vehicle boundaries and on line between target and turret
-                while( in_veh != veh_pointer_or_null( g->m.veh_at( path_to_target.back() ) ) ) {
```

Comment:
Keep using this method, it is *much* shorter than your new code.

Reply:
But Kevin wanted it to be changed. And what to do then?

Reply:
Than keep it as it is now.